README.md

```diff
@@ -141,30 +135,34 @@ some meaningful syntax in the source file. This option is enabled by default.
 To set up your development environment, perform the following steps:
 1. Install the newest Node.js version for your operating system (OS); see the [Node.Js homepage](https://nodejs.org/en/).
 2. Install a Git client for your OS; see the [Git homepage](https://git-scm.com/downloads).
-3. Install dependencies using `yarn install`, as superuser. (Use `Open command prompt (CMD) (admin)` on Windows.)
-   See the [Yarn homepage](https://yarnpkg.com/).
+3. Install dependencies using `npm install`, as superuser. (Use `Open command prompt (CMD) (admin)` on Windows.)
+    See the [npm Docs](https://docs.npmjs.com/cli/v8/commands/npm-install).
 4. Enable the corepack utility by running `corepack enable`, as superuser.
-   See the [Corepack homepage](https://github.com/nodejs/corepack).
+    See the [Corepack homepage](https://github.com/nodejs/corepack).
 
 To start working on this project, perform the following steps:
 1. Create a working directory for the project and change into it.
 2. Clone the project in the new directory `git clone https://github.com/eisopux/checker-framework-vscode.git`
-   (adapt the URL to your specific fork and branch). 
+    (adapt the URL to your specific fork and branch). 
 3. Change into the cloned directory `cd checker-framework-vscode`.
-4. Run `yarn global add vsce`.
+4. Run `npm install -g vsce`.
+
+Compile the downloader locally: 
+1. Create a working directory for the downloader and change into it. 
+2. Clone the project in the new directory `git clone https://github.com/eisopux/checker-framework-languageserver-downloader.git`
+3. Cd into the downloader folder and run `./gradlew assemble`
+4. Copy `build/libs/checker-framework-languageserver-downloader-all.jar` to the plugin’s folder and rename it to replace the file `checker-framework-languageserver-downloader-0.2.0.jar`
```

Comment:
In a separate PR, can you think about a nicer way how to handle this jar file?

Reply:
Will do! Zhiping had helped me compile the downloader locally, so I will reach out to him and see if we can figure it out.